api/src/main/java/club/mondaylunch/gatos/api/controller/NodeTypesController.java

```diff
@@ -12,10 +14,17 @@
 @RestController
 @RequestMapping("api/v1/node-types")
 public class NodeTypesController {
+    public static final Map<String, String> ENGLISH_DISPLAY_NAMES = getTranslatedDisplayNameMap("en_gb");
 
-    public record NodeTypeInfo(String name, String category) {
+    public record NodeTypeInfo(String name, String category, String displayName) {
```

Comment:
more things than just node types need display names. input + output connectors, settings, datatypes, enum values even, probably something else I'm forgetting. we can't wrap all of them like this.